src/Transpile.ts

```diff
@@ -1,11 +1,13 @@
 import * as ts from "typescript";
-import { CompilerOptions } from "./CompilerOptions";
+import { CompilerOptions, validateOptions } from "./CompilerOptions";
 import * as diagnosticFactories from "./diagnostics";
 import { Block } from "./LuaAST";
 import { LuaPrinter } from "./LuaPrinter";
 import { LuaTransformer } from "./LuaTransformer";
 import { TranspileError } from "./TranspileError";
 import { getCustomTransformers } from "./TSTransformers";
+import { bundleTranspiledFiles } from "./bundle";
+import { SourceNode } from "source-map";
```

Comment:
Can you run TypeScript's `Organize Imports` command (Ctrl+Shift+O in vscode)?

Reply:
Can we let prettier do this?

Reply:
Unfortunately now there's no good way to enforce that.
Prettier never does changes that may change something in runtime.
ESLint currently doesn't have a rule that follows the same order.
TSLint's [ordered-imports](https://palantir.github.io/tslint/rules/ordered-imports/) is close, but it only sorts them, while `Organize Imports` also removes unneeded ones, removes formatting and handles `export from`.
There's also an option to utilize original command with a [CLI](https://www.npmjs.com/package/organize-imports-cli), but I'm not sure if we want to add a new tool.